### PR TITLE
Fix/empty targetframework msbuild var

### LINKED ISSUE #2496: Paket fails on projects that target multiple frameworks
### Description

I made a very simple project that only depends on one package. It targets both .NET Standard 1.6 and .NET Framework 4.6.2. Running `paket install` and `paket restore` gives no errors. But when it run `dotnet restore`, Paket says this:

`C:\code\paket_bug\.paket\Paket.Restore.targets(22,5): error MSB3073: The command ""C:\code\paket_bug\.paket\paket.exe" restore --project "C:\code\paket_bug\Hello.fsproj" --target-framework " exited with code 1. [C:\code\paket_bug\Hello.fsproj]`

The target framework is empty!

### Repro steps

__hellp.fsproj__
``` xml
<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFrameworks>net462;netstandard1.6</TargetFrameworks>
  </PropertyGroup>
  <ItemGroup>
    <Compile Include="Library.fs" />
  </ItemGroup>
  <ItemGroup>
    <PackageReference Include="FSharp.NET.Sdk" Version="1.0.*" PrivateAssets="All" />
  </ItemGroup>
  <Import Project=".paket\Paket.Restore.targets" />
</Project>
```

__paket.dependencies__
```
framework: net462, netstandard1.6

source https://api.nuget.org/v3/index.json

nuget Chessie
```

__paket.references__
`Chessie`

### Possible cause

In the project file, there is a `TargetFrameworks` section, not the `TargetFramework` that `paket.restore.targets` asks for. `paket.restore.targets` should handle it somehow. On projects that target one framework and have the `TargetFramework` property, Paket works like a charm.

### Version information

__paket --version__: `Paket version 5.5.3`
__dotnet --version__:  `2.0.0-preview2-006497`

### PR DESCRIPTION
**Pull request by Mpdreamz**
Fixes #2496 where `$(TargetFramework)` returns an empty string because only `<TargetFrameworks>` is set.

### CODE REVIEW

File: .paket/Paket.Restore.targets [hunk 1]
```diff
@@ -19,7 +19,11 @@
   <Target Name="PaketRestore" BeforeTargets="_GenerateDotnetCliToolReferenceSpecs;_GenerateProjectRestoreGraphPerFramework;_GenerateRestoreGraphWalkPerFramework;CollectPackageReferences" >
 
     <Exec Command='$(PaketBootStrapperCommand) ' Condition="Exists('$(PaketBootStrapperExePath)') AND !(Exists('$(PaketExePath)'))" ContinueOnError="false" />
-    <Exec Command='$(PaketCommand) restore --project "$(MSBuildProjectFullPath)" --target-framework $(TargetFramework)' ContinueOnError="false" />
+    <Exec Command='$(PaketCommand) restore --project "$(MSBuildProjectFullPath)" --target-framework $(TargetFramework)' Condition="$(TargetFramework) != ''" ContinueOnError="false" />
+    <ItemGroup Condition="$(TargetFramework) == ''">
+      <SplitTFM Include="$(TargetFrameworks.Split(`;`))" />
```
Comment (forki) [inline .paket/Paket.Restore.targets]: 💋

**Comment by matthid** [comment]
Or we could add `--target-frameworks` to paket because there is probably no need to run paket multiple times...

**Comment by forki** [comment]
Yes. That would be even better

**Comment by forki** [comment]
closing in favor of https://github.com/fsprojects/Paket/pull/2526 which includes this one

**Pull request closed without merging**